### ISSUE TITLE
Does not work when the trigger event of the action is not a commit

### ISSUE THREAD

**Issue by Torwent** [Sniddl/discord-commits#22]
So I have a chain of github action that depend on each other.

I use this on my second file which is triggered with:
```yml
on:   
  workflow_run:
    workflows: ["Compile test"]
    branches: [master]
    types:
      - completed
```

And this doesn't work because of that.
The thing is, I want to use another way of getting my commit message and I can't because it won't even run

**Comment by ZebTheWizard** [comment]
Can you post an example of what "Compile test" looks like? Or link to the failed action you're referring to?

I added two extra workflows to the `v1.6` branch, and it seems run both just fine.

https://github.com/Sniddl/discord-commits/tree/v1.6/.github/workflows

**Comment by Torwent** [comment]
Well.... my "compile test" really shouldn't matter I think, but anyway, this is the specific compile test I am using in my project:

https://github.com/Torwent/WaspLib/blob/master/.github/workflows/compile-test.yml

After that runs successfully, the next action is run, which is where I was trying to use your action:
https://github.com/Torwent/WaspLib/blob/master/.github/workflows/bump-version.yml

the "discord notification" job is where I was trying to use your action, I ended up switching to something else and after a lot of trial and error managed to get it to work for me.
What was preventing me from getting the commit message was that I was using 
`github.event.head_commit.message` and in this case, to get access to the original commit that started of the workflow I needed to use `github.event.workflow_run.head_commit.message`.

I suspect that was the issue with your action, I didn't look into it's inner workings but I figure it's the same issue

**Comment by ZebTheWizard** [comment]
Currently this action only supports data provided by the `@actions/github` package or the `process.env` provided by NodeJS. 

More information about the context API can be found here: https://docs.github.com/en/actions/learn-github-actions/contexts

For specifically working with `github` context check out this page: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

I noticed that `@actions/github` is on version `2.2.0` and has since updated to version `5.1.1`.  I haven't really look into it either but it is very possible the workflow context was not included in earlier versions. This is entirely speculation on my part.

I've released `v1.6` which has the latest package updates. 

Please let me know if this wasn't helpful.
